### DISCUSSION TITLE
Graceful Cluster Scaling  & Reacting to alerts using (Prometheus + Cruise Control)

### DISCUSSION THREAD

**Discussion by ghost** [strimzi/strimzi-kafka-operator#7628] [Q&A]
**Is your feature request related to a problem? Please describe.**
Can we do cluster scaling based on alerts gathered from prometheus + cruise control

**Describe the solution you'd like**
[Similar to what is implemented in Banzai cloud ](https://banzaicloud.com/blog/kafka-alert/)

**Comment by scholzj** [comment]
Perhaps you should elaborate a bit more on what are you actually asking for. Once there is a clear description, then we can see if some issue for that exists or doesn't exist and open one.
